src/weights.jl

```diff
@@ -281,6 +290,9 @@ Base.mean{T<:Number,W<:Real}(A::AbstractArray{T}, w::WeightVec{W}, dim::Int) =
 
 
 ###### Weighted median #####
+function Base.median(v::AbstractArray, w::WeightVec)
+    throw(MethodError(median, (typeof(v), typeof(w))))
```

Comment:
Hadn't noticed that one. Should pass `(v, w)` instead of their types. Otherwise, looks good to me.